README.md

```diff
@@ -21,14 +26,21 @@ Several other items can be configured by setting environment variables:
 
 more are available, consult legacy_modules.py.
 
-Data
+#### Data
 Botiana loads local data from a yaml file: data/data.yaml
 
-Keywords & Reactions
+#### Keywords & Reactions
 Keywords, responses and reactions can occur in specific channels, or all channels. Configure this in the data.yaml file.
 
-Systems Administrator Dictionary
+#### Systems Administrator Dictionary
 Local dictionary entries can be created for lookup. Configure this in the data.yaml file.
 
 ### Build
-docker build . -t botiana
+`docker build . -t botiana`
+
+### Local testing
+Get your token from the slack api and export it as shown and run with docker.
+```
+export token="xoxb-11111111111-111111111111-aaa1aAAAAAaaaAAaAAAAaa1a"
```

Comment:
I tried this token, it doesn't work, plz put a working token here thnx

Reply:
replace the 1 with 3 and A with z. works on my machine ;)